index.d.ts

```diff
@@ -1,6 +1,6 @@
 export interface PluginOptions {
     configuration?: any;
-    formatter?: string;
+    formatter?: string | any;
```

Comment:
Should the type be `string | Function` - or even a more strictly defined function type?